src/logger.ts

```diff
@@ -0,0 +1,17 @@
+import { Logger, makeConsoleLogFactory } from "@graphile/logger";
+
+export { LogLevel as MigrateLogLevel } from "@graphile/logger";
+export { Logger as MigrateLogger } from "@graphile/logger";
+export { LogFunctionFactory as MigrateLogFactory } from "@graphile/logger";
+export interface MigrateLogMeta {
+  error?: Error;
+}
+
+export const defaultMigrateLogger = new Logger(
+  makeConsoleLogFactory({
+    format: `%s`,
+    formatParameters(_level, message) {
+      return [message];
+    },
+  }),
+);
```

Comment:
Even though this shares a name with @graphile/logger.defaultLogger I think it's fine for us to name it as `defaultLogger`; none of our other names are scoped with `Migrate`: 

```suggestion
export const defaultLogger = new Logger(
  makeConsoleLogFactory({
    format: `%s`,
    formatParameters(_level, message) {
      return [message];
    },
  }),
);
```